src/controller/BackPostController.php

```diff
@@ -166,7 +167,7 @@ public function deletePost(string $idPost, PostModel $postModel)
 		}
 
 		// display the confirm delete message
-		echo $this->twig->render('delete_post.twig', ['SESSION' => $_SESSION, 'post' => $post]);
+		$this->render('delete_post.twig', [], [], $_SESSION, $post);
```

Comment:
idem pour session

Reply:
Modif ok : 
$this->render('delete_post.twig', $_SESSION, [], [], $post);